fs/accounting/stats.go

```diff
@@ -248,7 +251,7 @@ func (s *StatsInfo) String() string {
 
 	s.mu.RLock()
 
-	dt := s.totalDuration()
+	dt := time.Since(startTime)
```

Comment:
We need this dt to be `s.totalDuration()` as it is used for the calculation of the overall speed.

So we we probably need a `elapsedTime := time.Since(startTime)` for use in the stats logging.